### PR TITLE
Deploy blackbox_prober

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-1266

Screen Shot 2020-06-17 at 4 07 00 PM

Require https://github.com/openshift/release/pull/9402

/cc @alvaroaleman @stevekuznetsov

### CODE REVIEW

**Comment by hongkailiu** [comment]
Alert will be follow up PRs.

**Comment by hongkailiu** [comment]
The metric is there:

https://prometheus-prow-monitoring.apps.ci.l2s4.p1.openshiftapps.com/graph?g0.range_input=1h&g0.expr=probe_ssl_earliest_cert_expiry&g0.tab=1

File: clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: additional-scrape-configs
+  namespace: prow-monitoring
+stringData:
+  prometheus-additional.yaml: |
+    - job_name: blackbox
```
Comment (alvaroaleman) [inline clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml:8]: Why can we not use the prometheus operator custom resources for this?

Comment (hongkailiu) [reply clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml:8]: https://github.com/kubernetes/test-infra/pull/15000#issuecomment-547189270

Comment (alvaroaleman) [reply clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml:8]: Can you add a comment with a link to https://github.com/coreos/prometheus-operator/issues/256 ?

Comment (hongkailiu) [reply clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml:8]: Done.

**Comment by hongkailiu** [comment]
/hold

Need to merge https://github.com/openshift/release/pull/9402 first.

File: clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      metrics_path: /probe
+      params:
+        module: [http_2xx]
+      static_configs:
+        - targets:
+          - https://registry.build02.ci.openshift.org
```
Comment (stevekuznetsov) [inline clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml:14]: We might as well add in the API, too, right? Anything we self-manage

Comment (hongkailiu) [reply clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml:14]: Yes.
This is POC.
I will add all of them later (when we are done with the their certs):

Prow:
- deck
- bokos
...

build01:
- prometheus
- registry

build02:
...

**Pull request merged**